catroidTest/src/org/catrobat/catroid/test/utils/StatusBarNotificationManagerTest.java

```diff
@@ -81,6 +83,18 @@ public void testShowOrUpdateNotification() {
 		}
 	}
 
+	public void testUploadRejectedNotification(){
```

Comment:
Missing Whitespace!
@robertpainsi : As far as i remember should this check be already in our ruleset !?